kubernetes/reporter/src/main/java/org/arquillian/cube/kubernetes/reporter/TakeKubernetesResourcesInformation.java

```diff
@@ -0,0 +1,175 @@
+
+        for (Service service : kubernetesClient.services().inNamespace(namespace).list().getItems()) {
+            TableRowEntry tableRowEntry = new TableRowEntry();
+            tableRowEntry.addCells(
+                    new TableCellEntry(service.getMetadata().getName()),
+                    new TableCellEntry(String.valueOf(service.getSpec().getClusterIP())),
+                    new TableCellEntry(getPortsForService(service)));
+
+            tableEntry.getTableBody().getRows().add(tableRowEntry);
+        }
+
+        return tableEntry;
+    }
+
+    private TableEntry getTableForPods(KubernetesClient kubernetesClient, String namespace) {
+
+        TableEntry tableEntry = createTableEntryWithTHead(POD, POD, STATUS);
+
+        for (Pod pod : kubernetesClient.pods().inNamespace(namespace).list().getItems()) {
+            TableRowEntry tableRowEntry = new TableRowEntry();
+            tableRowEntry.addCells(new TableCellEntry(pod.getMetadata().getName()), new TableCellEntry(pod.getStatus().getPhase()));
+            tableEntry.getTableBody().getRows().add(tableRowEntry);
+
+        }
+
+        return tableEntry;
+    }
+
+    private TableEntry createTableEntryWithTHead(String name, String... heads) {
+        TableEntry tableEntry = new TableEntry();
+        tableEntry.setTableName(name);
+
+        for (String head : heads) {
+            tableEntry.getTableHead().getRow().addCell(new TableCellEntry(head));
+        }
+
+        return tableEntry;
+    }
+
+    private FileEntry getFileForResourcesConfiguration(Configuration configuration, ReporterConfiguration reporterConfiguration) throws IOException {
+        String path = configuration.getEnvironmentConfigUrl().getPath();
+
+        final Path rootDir = Paths.get(reporterConfiguration.getRootDir().getAbsolutePath());
+        final Path relativize = rootDir.relativize(Paths.get(path));
+
+        FileEntry fileEntry = new FileEntry();
+        fileEntry.setPath(relativize.toString());
+        fileEntry.setMessage(CONFIGURATION);
+        fileEntry.setType(getFileType(relativize));
+
+        return fileEntry;
+    }
+
+    private KeyValueEntry getKeyValueEntry(String key, String value) {
+        return new KeyValueEntry(key, value);
+    }
+
+    private String getPortsForService(Service service) {
+        StringBuilder sb = new StringBuilder();
+
+        for (ServicePort servicePort : service.getSpec().getPorts()) {
+            sb.append(servicePort.getPort()).append(" ");
+        }
+
+        return sb.toString();
+    }
+
+    private void addEntry(GroupEntry groupEntry, PropertyEntry propertyEntry) {
+        groupEntry.getPropertyEntries().add(propertyEntry);
+    }
+
+    private String getFileType(Path path) throws IOException {
+        String extension = FilenameUtils.getExtension(path.toString());
+        String type = "";
+        if ("json".equals(extension)) {
+            type = Files.probeContentType(path);
```

Comment:
Do we really have to `probe`? We have total control and knowledge at this point what we are passing along, so I'm not sure if we really have to check if we are dealing with json?